### PR TITLE
NAPSSPO-1047 - tssc-containers - rename tssc-tool-buildah to tssc-tool-containers and include podman, buildah, and skopeo

### PR DESCRIPTION
**Pull request by itewk**
# purpose
make one tools container image that has buildah/podman/skopeo rather then just buildah.

NOTE: still keeping the skopeo image on its own since it can be installed without private repos.

# manual images

* https://quay.io/repository/tssc/tssc-tool-containers?tab=tags (pr-33)
* https://quay.io/repository/tssc/tssc-tool-openscap?tab=tags (pr-33)

# integration tests
https://jenkins-jenkins.apps.tssc.rht-set.com/job/tssc-references/job/reference-quarkus-mvn-jenkins/view/change-requests/job/PR-17/3/

### CODE REVIEW

File: tssc-tool-containers/Dockerfile.ubi8 [hunk 1]
```diff
@@ -7,21 +7,25 @@ FROM $FROM_IMAGE
...
 
 # NOTE This image must run priviliged
 USER 0
 
 # Don't include container-selinux and remove
 # directories used by yum that are just taking
 # up space.
-RUN dnf -y module enable container-tools:rhel8; dnf -y update; dnf -y reinstall shadow-utils; dnf -y install buildah fuse-overlayfs /etc/containers/storage.conf; rm -rf /var/cache /var/log/dnf* /var/log/yum.*
+RUN dnf -y module enable container-tools:rhel8; \
+    dnf -y update; \
```
Comment (kfrankli) [inline tssc-tool-containers/Dockerfile.ubi8:25]: Yay readability!

**Review by kfrankli — APPROVED** [review]
LGTM

File: tssc-tool-containers/Dockerfile.ubi8 (hunk 1 above)
Comment (itewk) [reply tssc-tool-containers/Dockerfile.ubi8:25]: :)

Comment (itewk) [reply tssc-tool-containers/Dockerfile.ubi8:25]: now approve

**Review by HunterGerlach — APPROVED** [review]
Looks good!

**Comment by itewk** [comment]
manual images built:
* tssc-tool-containers - https://quay.io/repository/tssc/tssc-tool-containers?tab=history
* tssc-tool-openscap - https://quay.io/repository/tssc/tssc-tool-openscap?tab=history

**Pull request merged**